lychee-bin/src/main.rs

```diff
@@ -113,6 +115,18 @@ fn run_main() -> Result<i32> {
     if let Some(c) = Config::load_from_file(&opts.config_file)? {
         opts.config.merge(c)
     }
+
+    // Load excludes from file
+    for path in &opts.config.exclude_file {
+        let file = File::open(path).expect("No such file");
+        opts.config.exclude.append(
+            &mut io::BufReader::new(file)
+                .lines()
+                .map(|l| l.expect("Could not read line"))
```

Comment:
there's no need to manually unwrap a Result in Iterator.
If you've a Itertor<Item = Result<T, E>>, you can collect it directly to a Result<Vec<T>, E>
- If there's any Err, it returns an Err
- If all elements arr Ok, it returns Ok(Vec<T>)

Reply:
@dblock I resolved this by pushing to your branch if you want to have a look at the diff.